diff --git a/SpringbootSchoolShop/src/main/java/com/zhang/ssmschoolshop/controller/admin/ExcelController.java b/SpringbootSchoolShop/src/main/java/com/zhang/ssmschoolshop/controller/admin/ExcelController.java
index 1aaf420..c4f5060 100644
--- a/SpringbootSchoolShop/src/main/java/com/zhang/ssmschoolshop/controller/admin/ExcelController.java
+++ b/SpringbootSchoolShop/src/main/java/com/zhang/ssmschoolshop/controller/admin/ExcelController.java
@@ -4,7 +4,6 @@ import com.zhang.ssmschoolshop.entity.Goods;
 import com.zhang.ssmschoolshop.entity.GoodsExample;
 import com.zhang.ssmschoolshop.service.GoodsService;
 import com.zhang.ssmschoolshop.util.ExcelUtils;
-import org.apache.poi.ss.formula.functions.T;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
@@ -30,23 +29,21 @@ public class ExcelController {
 
 
     @GetMapping("/excel/export")
-    public String ExportRecord(HttpServletResponse response, HttpServletRequest request) {
+    public void ExportRecord(HttpServletResponse response, HttpServletRequest request) {
         List<Goods> goodsList = goodsService.selectByExample(new GoodsExample());
 
         //定义导出的excel名字
         String excelName = "资源详情表";
 
         //导出用户相关信息
-        new ExcelUtils().export(excelName, goodsList, Goods.class, response);
-        return "导出资源成功";
+        ExcelUtils.export(excelName, goodsList, Goods.class, response);
     }
 
     @GetMapping("/excel/import")
-    public String ImportRecord(HttpServletResponse response, HttpServletRequest request) {
+    public void ImportRecord(HttpServletResponse response, HttpServletRequest request) {
         String path = "D:\\github\\myproject\\SptingbootSchoolShop\\import.txt";
         //List<Goods> goodsList=(List<Goods>)ExcelUtils.getRecordByTxt(path);
         // 入库
-        return "导入资源成功";
     }
 
 
